components/styled/table/src/molecules/questionnaire-table.tsx

```diff
@@ -4,8 +4,10 @@ import {
   SummaryTableViewType,
   QuestionnaireResponseItem,
   QuestionnaireResponseItemAnswer,
+  Maybe,
 } from '@ltht-react/types'
 import { EnsureMaybe, EnsureMaybeArray, partialDateTimeText } from '@ltht-react/utils'
+import { Checkbox } from '@material-ui/core'
```

Comment:
Oh I should say - we use FontAwesome for our icons rather than Material UI. There's a list of them in `components/icon/src/atoms` 
I'm making a few changes to them in my branch though so don't worry too much